Configuration/Eras/python/Modifier_run3_miniAOD_12X_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
@simonepigazzini can you please add a description here, for posterity

Reply:
indeed